sources/packages/backend/apps/api/src/route-controllers/student-loan-balance/models/student-loan-balance.dto.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Student loan balance details of a month.
+ */
+class StudentMonthlyLoanBalanceDetailAPIOutDTO {
+  balanceDate: string;
+  cslBalance: number;
+}
+
+/**
+ * Monthly loan balance details of a student
+ * upto 12 months from current date.
```

Comment:
Minor, not a blocker, I am not sure if "upto 12 months from current date." is meaningful to be added at DTO level.